queries.sql

```diff
@@ -64,4 +64,51 @@ FROM
   vet_clinic
 WHERE
   weight_kg >= 10.4
-  AND weight_kg <= 17.3;
\ No newline at end of file
+  AND weight_kg <= 17.3;
+
+SELECT
+  COUNT(name)
+FROM
+  animals;
+
+SELECT
+  COUNT(name)
```

Comment:
* Kindly add queries for all your transactions i see you added a screenshot but your queries don't give the same result as no single query was written to handle transactions